clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -1068,6 +1069,23 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
    */
   DeleteGroupCommandStep1 newDeleteGroupCommand(long groupKey);
 
+  /**
+   * Command to assign a user to a group.
+   *
+   * <pre>
+   *
+   *
+   * zeebeClient
+   *  .newAssignUserToGroupCommand(123L, 456L)
+   *  .send();
+   * </pre>
+   *
+   * <p>This command is only sent via REST over HTTP, not via gRPC <br>
+   *
+   * @return a builder for the command
+   */
+  AssignUserToGroupCommandStep1 newAssignUserToGroupCommand(long userKey, long groupKey);
```

Comment:
❌ We should try and keep this aligned with the API of assigning a user to a tenant. Here we use:

```java
zeebeClient
  .newAssignUserToTenantCommand(tenantKey)
  .userKey(userKey)
  .send();
```

Reply:
@remcowesterhoud, it seems like we're misaligned in several places. The `AssignGroupToTenantCommand` implementation is similar to this one (it was like that from my feedback). 

In general, it made more sense to me, as both `group` and `user` key are required parameteres. But I agree with you, the general pattern is that we pass along the key of the "entity" we update in the contractor. All other parameters are passed along in "builder" methods.

So I'll update this one. And I'll try to update the group-to-tenant un/assignment commands as well.